packages/apps/crm/src/screens/event/EventPlanningScreen.js

```diff
@@ -0,0 +1,259 @@
+
+function EventPlanningScreen({navigation}) {
+  const dispatch = useDispatch();
+  const Colors = useThemeColor();
+  const I18n = useTranslator();
+  const {eventList, loading} = useSelector(state => state.event);
+  const {userId} = useSelector(state => state.auth);
+  const [filteredList, setFilteredList] = useState(eventList);
+  const [assigned, setAssigned] = useState(false);
+  const [selectedStatus, setSelectedStatus] = useState([]);
+  const [plannedEvent, setPlannedEvent] = useState(null);
+  const commonStyles = useMemo(() => getCommonStyles(Colors), [Colors]);
+  const [dateSave, setDateSave] = useState(null);
+
+  useEffect(() => {
+    setFilteredList(filterOnUserAssigned(filterOnStatus(eventList)));
+  }, [
+    filterOnUserAssigned,
+    filterOnStatus,
+    eventList,
+    selectedStatus,
+    assigned,
+  ]);
+
+  const listItem = useMemo(() => {
+    return EventType.getCalendarListItems(filteredList, Colors);
+  }, [Colors, filteredList]);
+
+  const fetchEventFilter = useCallback(
+    searchValue => {
+      dispatch(fetchPlannedEvent({date: dateSave, searchValue: searchValue}));
+    },
+    [dispatch, dateSave],
+  );
+
+  const filterOnStatus = useCallback(
+    list => {
+      return filterChip(list, selectedStatus, 'typeSelect');
+    },
+    [selectedStatus],
+  );
+
+  const styles = useMemo(() => {
+    return getStyles(Colors);
+  }, [Colors]);
+
+  const fetchItemsByMonth = useCallback(
+    date => {
+      dateSave === null && setDateSave(date);
+      dispatch(fetchPlannedEvent({date: date}));
+    },
+    [dispatch, dateSave],
+  );
+
+  const navigateToEvent = id => {
+    if (id != null) {
+      navigation.navigate('EventDetailsScreen', {
+        eventId: id,
+      });
+    }
+  };
+
+  const rendBorderColor = borderColor => {
+    return {
+      borderLeftWidth: 7,
+      borderLeftColor: borderColor,
+    };
+  };
+
+  const renderDayEventDetails = ({id, data: event}) => {
+    if (event == null) {
+      return null;
+    }
+    return (
+      <TouchableOpacity activeOpacity={0.9} onPress={() => navigateToEvent(id)}>
+        <Card
+          key={id}
+          style={[styles.container, rendBorderColor(event.border)]}>
+          <Text style={styles.bold}>{event.subject}</Text>
+          {event.contactPartner && (
+            <LabelText iconName="map-pin" title={event.contactPartner} />
+          )}
+          {event.location && (
+            <LabelText iconName="map-pin" title={event.location} />
+          )}
+        </Card>
+      </TouchableOpacity>
+    );
+  };
+
+  const renderDayEvent = ({id, data: event}) => {
+    if (event == null) {
+      return null;
+    }
+
+    return (
+      <TouchableOpacity activeOpacity={0.9} onPress={() => navigateToEvent(id)}>
+        <Card
+          key={id}
+          style={[styles.container, rendBorderColor(event.border)]}>
+          <Text style={styles.bold}>{event.subject}</Text>
+        </Card>
+      </TouchableOpacity>
+    );
+  };
```

Comment:
Those two functions render the same card with two optional fields, I think we can simplify by creating a PlanningEventCard component with a boolean as props to know if location and contactPartner should be rendered or not.